components/camel-salesforce/camel-salesforce-component/src/main/java/org/apache/camel/component/salesforce/SalesforceProducer.java

```diff
@@ -93,6 +96,15 @@ private boolean isAnalyticsOperation(OperationName operationName) {
         }
     }
 
+    private boolean isCompositeOperation(OperationName operationName) {
+        switch (operationName) {
+        case COMPOSITE_TREE:
+            return true;
+        default:
+            return false;
+        }
+    }
+
```

Comment:
yes, but the it gets added to in https://github.com/apache/camel/pull/1344/files#diff-bad25da55069bde3a6dab543a106b505R99 to include COMPOSITE_BATCH as well, I knew this before hand so I put a switch/case there